src/common/test-helpers/button.helper.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { Button } from "@map-colonies/react-core";
+import { ReactWrapper, ShallowWrapper } from "enzyme";
+
+/* eslint-disable */
+export const getButtonById = (wrapper: ReactWrapper | ShallowWrapper, id: string): ReactWrapper | ShallowWrapper => {
+  return wrapper
+    // @ts-ignore
+    .findWhere((n) => {
+      return n.type() === Button &&
+        n.prop('children').props['id'] === id;
+    });
+};
+
+describe.skip('Button helpers dummy suite', () => {
+  it('dummy test', () => {
```

Comment:
what is the purpose of this test?

Reply:
It's for store helpers in *.spec files and not include them in build.